src/helpers/translations/translations.js

```diff
@@ -0,0 +1,19 @@
+export default {
+  locale: "en",
+  crypto: {
+    ethereum: "etherereum",
+    tokens: "tokens",
+    nft: "Non-Fungible Token",
+    whatIsANFT:
+      " is a way to ensure digital assets are unique and easily tradable on a blockchain.",
+    harbergerTax: "Harberger Tax"
+  },
+  muggle: {
+    ethereum: "blockchain",
+    tokens: "animals",
+    nft: "Crypto Collectable",
```

Comment:
Not crazy about crypto collectible. Also not sure on spelling of "Collectable".  Would prefer blockchain animal, or blockchain creature?

Reply:
Cool, went with blockchain animal. Wasnt so hot on collectable either. South African spelling is with an A and US spelling is with an i.